rpc/backend/backend.go

```diff
@@ -196,7 +196,7 @@ func (b *EthermintBackend) PendingTransactions() ([]*rpctypes.Transaction, error
 		return nil, err
 	}
 
-	transactions := make([]*rpctypes.Transaction, pendingTxs.Count)
+	transactions := make([]*rpctypes.Transaction, 0)
 	for _, tx := range pendingTxs.Txs {
```

Comment:
you could leave the `transactions := ...` line the same as before, then change the `for` to be `for i, tx := ...` and assign to `transactions[i]` instead of appending